dcos_test_utils/package.py

```diff
@@ -0,0 +1,88 @@
+"""Utilities for integration testing package management (https://github.com/dcos/cosmos)
+"""
+
+import logging
+
+from dcos_test_utils.helpers import (ApiClientSession,
+                                     RetryCommonHttpErrorsMixin)
+
+log = logging.getLogger(__name__)
+
+
+class Cosmos(RetryCommonHttpErrorsMixin, ApiClientSession):
+    def __init__(self, default_url, default_os_user='root', session=None):
+        super().__init__(default_url)
+        if session is not None:
+            self.session = session
+        self.default_os_user = default_os_user
```

Comment:
default_os_user is a thing for marathon. I think you should be able to completely remove it in this module